unitex/src/fr/umlv/unitex/leximir/helper/DelasHelper.java

```diff
@@ -80,17 +80,21 @@ else if(dic!=null){
         for(String dela:list){
             //String path = Utils.getValueXml("pathDelas")+"/"+dela;
             String path="";
-            if(allDelas)
-            	path = DictionaryPath.allDelas+File.separator+dela;
-            else
+            if(allDelas){
+            	path = DictionaryPath.allDelas+File.separator + dela;
```

Comment:
All binary operators except `.` should be separated from their operands by spaces